### PR TITLE
[add] py-metomi-rose: new recipe, required by py-cylc-rose

### PR DESCRIPTION
**Pull request by LydDeb**

### CODE REVIEW

**Review by tldahlgren — COMMENTED** [review]
Confirmed the homepage and version sha256.  Adding python package maintainer.

File: var/spack/repos/builtin/packages/py-metomi-rose/package.py
Comment (manuelakuhn) [inline var/spack/repos/builtin/packages/py-metomi-rose/package.py]: Optional: spack only supports python@3.7: anyway, so this dependency is not needed.

**Review by manuelakuhn — DISMISSED** [review]
Test will only run through once #39982 and #39990 are merged.

**Review by adamjstewart — DISMISSED** [review]

**Review by manuelakuhn — APPROVED** [review]

**Review by adamjstewart — APPROVED** [review]

**Pull request merged**
